src/styles/blocks/header.scss

```diff
@@ -0,0 +1,185 @@
+.header {
+  height: 100vh;
+  background-image: url(../styles/images/pictures/Cover_photo.png);
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+
+  @include for-phone {
+    background-image: url(../styles/images/pictures/Cover_photo_phone.png);
+    background-size: cover;
+  }
+
+  &__bar {
+    height: 100px;
+    max-width: 1020px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 0 auto;
+    padding: 36px;
+
+    @include for-tablet {
+      max-width: 600px;
+      height: 50px;
+      padding: 25px;
+    }
+
+    @include for-phone {
+      height: 30px;
+      justify-content: center;
+    }
+  }
+
+  &__logo {
+    @include flex-center;
+    &-img {
+      @include for-tablet {
+        width: 130px;
+      }
+    }
+
+  }
+
+  &__text {
+    color: white;
```

Comment:
You use this value several times, better to create a variable. And maybe it is better to write it in hex format as your other colors written using hex.

Reply:
Fixed